docs/visual-basic/reference/vb6-support.md

```diff
@@ -0,0 +1,349 @@
+
+   Key Visual Basic 6.0 runtime files, used in the majority of application scenarios, are shipping in and supported for the lifetime of supported Windows versions. This lifetime is five years of mainstream support and five years of extended support from the time that a given version of Windows ships. These files have been tested for compatibility as part of our testing of Visual Basic 6.0 applications running on supported Windows versions. 
+
+   > [!NOTE]
+   > All supported Windows versions contain a nearly identical list of files, and the redist requirements for applications containing these files should be nearly identical. One key difference is that `TriEdit.dll` was removed from Windows Vista and later versions.
+
+- Supported runtime files –- Extended files to distribute with your application
+
+   This extended list consists of key controls, libraries, and tools that are installed from the IDE media or from Microsoft.com to the developer machine. Typically, the VB6 IDE installed these controls to the developer machine by default. The developer still needs to redistribute these files with the application. The supported version of the files is available online on the Microsoft Download Center (http://go.microsoft.com/fwlink/?LinkID=142927).
+
+- Unsupported runtime files
+
+   Some files either have fallen out of mainstream support or were never included as a part of the runtime redist (e.g., they were included in the `\Tools` folder on the IDE media to support legacy VB4/VB5 applications, or they were third-party controls). These files are not supported on Windows; instead they are subject to whatever support agreement applies to the media they were shipped with. This implies no warranties around support and servicing. In some instances, later versions of these libraries are supported. Details on backward compatibility or migration to supported versions are provided below.
+
+For specific details on the files included in each support group see the [Runtime Definition](#runtime-definition) section below.
+
+## The Visual Basic 6.0 support lifetime
+
+Supporting and/or shipping Visual Basic 6.0 runtime binaries on supported Windows versions does not change the support policy for the Visual Basic 6.0 IDE or Visual Studio 6.0 IDE as a whole. Those products moved out of extended support on April 8, 2008.
+
+~Further details on the support policy for the Visual Basic 6.0 and Visual Studio 6.0 products can be found at http://msdn.microsoft.com/en-us/vstudio/aa718686.aspx.~ Details on the support lifecycle of Microsoft products can be found at http://support.microsoft.com/gp/lifepolicy. As a part of this support lifecycle, Microsoft will continue to support the Visual Basic 6.0 runtime on supported Windows versions for the support lifetime of those operating systems. This means, for example, that the Visual Basic 6.0 runtime will be supported on Windows Server 2003 until June, 2008 for Mainstream Support and June, 2013 for Extended Support.
```

Comment:
The link is broken. I thought I'd leave it here as crossed-out text until we're ready to merge, just in case we can figure out what the document the link should refer to.